client/packages/common/src/authentication/api/hooks/useLogin.ts

```diff
@@ -139,7 +139,7 @@ export const useLogin = (
       store,
       token,
       user: {
-        id: '',
+        id: userDetails?.userId ?? '',
```

Comment:
@mark-prins I thought its better to investigate why the user id was empty... Was this on purpose? is there a reason that it wasn't set?

Reply:
no specific reason - iirc the id was not returned by the API initially and I think the omission has remained in the front end code ever since